DocSum/docker_compose/amd/gpu/rocm/README.md

```diff
@@ -241,11 +241,10 @@ curl http://${HOST_IP}:${DOCSUM_BACKEND_SERVER_PORT}/v1/docsum \
 
 ### Query with audio and video
 
-> Audio and Video file uploads are not supported in docsum with curl request, please use the Gradio-UI.
```

Comment:
Maybe there could be a short blurb here saying something like: Audio and video can be passed as base64 strings or uploaded by providing a local file path. 

A new user reading through this may not immediately realize that curl commands in the snippets are basically equivalent and we're just showing different ways of doing the same thing.